tests/integration_tests_control_files/flow_solver/unsteady_sd7003.prm

```diff
@@ -0,0 +1,60 @@
+# Listing of Parameters
+# ---------------------
+# Number of dimensions
+
+set dimension = 3
+set run_type = flow_simulation
+set pde_type = navier_stokes
+
+# DG formulation
+set use_weak_form = true
+# set use_collocated_nodes = true
+
+# numerical fluxes
+set conv_num_flux = roe
+set diss_num_flux = symm_internal_penalty
+
+# ODE solver
+subsection ODE solver
+  set ode_output = quiet
+  set ode_solver_type = explicit
+  set output_solution_every_x_steps = 1
```

Comment:
I agree with Pranshul's comment. Also, I replaced the `explicit` ode type with `runge_kutta` in a PR last year -- this will need to be updated before this test can run!